test/test/util/manifest_generator.js

```diff
@@ -416,11 +429,12 @@ shaka.test.ManifestGenerator.prototype.isIdUsed_ = function(id) {
  * @param {number} id
  * @param {string} type
  * @param {string} language
+ * @param {?string} label
  * @return {!shakaExtern.Stream}
  * @private
  */
 shaka.test.ManifestGenerator.prototype.createStream_ =
-    function(id, type, language) {
+    function(id, type, language, label) {
```

Comment:
Since you have the `label()` method to set the label of the current stream, I think we don't need to add this parameter.